FBC/FunctionalEntities/RegulatoryAgencies.rdf

```diff
@@ -255,6 +259,21 @@
 		<skos:definition>a service provided by a regulatory agency, which may include, but not be limited to, examination, monitoring, supervision, testing, or other capabilities required to ensure the integrity, fairness, safety, or other capacity of a given industry, organization, or product</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fbc-fct-rga;TaxAuthority">
```

Comment:
Also need the Scheme and the Jurisdiction.

Reply:
done - took the same approach we used for rating scales / schemes in ratings to link the tax authority to the scheme